### ISSUE TITLE
JS Intellisense accepting suggestion when pressing period key

### ISSUE THREAD

**Issue by cvocke** [Microsoft/vscode#60714]
Issue Type: <b>Bug</b>

Steps to reproduce

1. Open a .js file
2. Trigger Intellisense with a period. i.e. `coolVariable.`
3. Press period once more. The suggestion is accepted and appended to the code

According to the docs only tab and enter should accept suggestions: https://code.visualstudio.com/docs/editor/intellisense#_settings

I often use js spread syntax `...` within arrays and objects, which is how I initially noticed this issue after updating to the most recent version. Though it appears to happen regardless of the type or location of the suggestion.

Here's a couple examples, I'm pressing the period key here, not tab or enter. I've tried moving small code snippets to new files and the issue persists. 

![js string intellisense bug](https://user-images.githubusercontent.com/29632566/46843453-b07c1100-cd87-11e8-9a13-70ec854322ab.gif)

![js string intellisense bug 2](https://user-images.githubusercontent.com/29632566/46843456-b4a82e80-cd87-11e8-8628-e748377a15a7.gif)

VS Code version: Code 1.28.1 (3368db6750222d319c851f6d90eb619d886e08f5, 2018-10-11T18:13:53.910Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-3770 CPU @ 3.40GHz (8 x 3392)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: unavailable_software<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|23.96GB (14.63GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details>Extensions: none

**Comment by egamma** [comment]
The ´.´ is now considered a trigger character that accepts a proposal, so this is the intentional behaviour.

**Comment by cvocke** [comment]
I did some further testing and the main issue I'm having with this behavior is when using spread syntax `...`. I found that I'm only having suggestions being made when period is pressed on a new line in multi-line arrays or objects. Single line arrays and objects are fine. Checkout the difference in behavior:

![multiline array behavior](https://user-images.githubusercontent.com/29632566/46884654-76a01e80-ce0a-11e8-80c6-6f728738baed.gif)

So perhaps this is a matter of tuning the JS Intellisense so that it isn't making suggestions after a single period that's on a new line in a multi-line array or object? If that was fixed I could see myself welcoming this behavior as it would help in many cases.

I'd also propose adding a setting for turning this behavior on or off - just like `editor.acceptSuggestionOnEnter`. I haven't looked at the source but I would imagine it would be a quick addition since the setting has already been implemented for Enter. The docs should also be updated. I can open up another issue if need be.
